src/db12/benchmark.py

```diff
@@ -0,0 +1,158 @@
+"""Contains all the functions needed to run DB12"""
+
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import os
+import sys
+import random
+import multiprocessing
+
+if sys.version_info[0] >= 3:
+    #pylint: disable = E, W, R, C
+    long = int 
+    xrange = range
```

Comment:
TBH I would go the other way around

Reply:
What do you mean exactly?

Reply:
I mean that I would code py3 style, and apply the changes `if sys.version_info[0] < 3:`